Library/Homebrew/requirements/macos_requirement.rb

```diff
@@ -0,0 +1,29 @@
+require "requirement"
+
+class MacOSRequirement < Requirement
+  fatal true
+
+  def initialize(tags = [])
+    @version = MacOS::Version.from_symbol(tags.first) unless tags.empty?
+    super
+  end
+
+  def minimum_version_specified?
+    OS.mac? && @version
+  end
+
+  satisfy(build_env: false) do
+    next OS.mac? || @version unless minimum_version_specified?
```

Comment:
Sorry, last nite: can you split this into two `next` checks and return a boolean value?

Reply:
Done.